magnusrodseth/systemsoft
# Update/next #22

**TODO**

- TypeScript is not too much of a fan of the new change, but this is simply a linting issue. When we find out how to import png's without getting red squiggly lines, we're good to update to Next.js 11 🚀

**Changelog**

- Update to Next.js 11
- Fixed all linting issues occurring

**How to test**

- Navigate to `frontend`
- Run `yarn`
- Run `yarn lint`
- Observe that there are no lint errors
